### PR TITLE
Changelog entries: PHP 5, PHP 7.0

### PR DESCRIPTION
**Pull request by saundefined**
Removed changelog entries from:

- PHP 5.x
- PHP 7.0

### CODE REVIEW

**Comment by kamil-tekiela** [comment]
I can understand PHP 5 but why did you remove PHP 7? Did something change recently in the official guidelines?

File: reference/misc/functions/unpack.xml [hunk 1]
```diff
@@ -91,25 +91,6 @@
...
-         The "a" code now retains trailing NULL bytes.
-        </para>
-        <para>
-         The "A" code now strips all trailing ASCII whitespace (spaces, tabs,
-         newlines, carriage returns, and NULL bytes).
-        </para>
-        <para>
-         The "Z" code was added for NULL-padded strings, and removes trailing
-         NULL bytes.
-        </para>
```
Comment (cmb69) [inline reference/misc/functions/unpack.xml:110]: I'm not sure whether this info is available elsewhere in the manual.

File: reference/spl/splfileobject/getcsvcontrol.xml [hunk 2]
```diff
@@ -48,12 +48,6 @@
         The escape character can now be an empty string.
        </entry>
       </row>
-      <row>
-       <entry>5.6.25, 7.0.10</entry>
```
Comment (cmb69) [inline reference/spl/splfileobject/getcsvcontrol.xml:52]: We should keep this entry as well (for 7.0.10 only).

File: reference/xmlreader/xmlreader/getattributeno.xml [hunk 3]
```diff
@@ -43,31 +43,6 @@
...
-       <entry>&Description;</entry>
-      </row>
-     </thead>
-     <tbody>
-      <row>
-       <entry>5.6.0</entry>
-       <entry>
-        <methodname>XMLReader::getAttributeNo</methodname> now returns &null;
-        if the attribute doesn't exist.
-       </entry>
```
Comment (cmb69) [inline reference/xmlreader/xmlreader/getattributeno.xml:63]: Could you please also update the return value section which explicitly mentions this change?

**Review by cmb69 — COMMENTED** [review]
Thank you! Looks mostly fine to me, except for some issues.

@kamil-tekiela, we remove 7.0.0 changelog entries for quite a while.

**Comment by saundefined** [comment]
@cmb69 thank you! Fixed

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**